shuffle/cli/src/main.rs

```diff
@@ -47,8 +46,9 @@ pub enum Subcommand {
         account_key_path: PathBuf,
     },
     #[structopt(about = "Create a new account")]
+    #[structopt(about = "Create a new account")]
     Account {
-        project_path: PathBuf,
-        account_key_path: PathBuf,
```

Comment:
is it part of the project plan to remove these and use a default path in the home dir? that kinda makes it not so great ergonomically, since I can't be working off multiple networks.

Reply:
Yes for now we're going to use the default path in home dir. In the future as we're developing node_configs for trove/other networks, we'll add flags so users can specify the specific network they want to use (i.e. shuffle -- node -- blockchain trove)

Reply:
@davidiw There are also some change in approaches that make using a `~/.shuffle` more ergonomic. Specifically that `trove` is moving away from the idea of being their own custom genesis, and becoming a library that sits on the common Diem Core Framework genesis. And with the DPN not being a near term goal, this means for the quarter and perhaps half we may only have one genesis: Diem Core Framework (aka stdlib).